src/docfx/build/Build.cs

```diff
@@ -83,14 +97,14 @@ private static Task BuildOneFile(Context context, Document file, TableOfContents
                 case ContentType.TableOfContents:
                     return BuildTableOfContents.Build(context, file, buildChild);
                 default:
-                    return Task.CompletedTask;
+                    return Task.FromResult<DependencyMap>(null);
```

Comment:
use `DependencyMap.Empty` over `null`

Reply:
Empty will create a new instance

---
In reply to: [193674269](https://github.com/dotnet/docfx/pull/2859#discussion_r193674269) [](ancestors = 193674269)